src/components/siteSelection/SiteSelection.vue

```diff
@@ -19,7 +19,7 @@
             :aria-label="$t('$sitePlaceholder')"
           >
             <option :value="0" disabled hidden>
-              {{ $t('$sitePlaceholder') }}
+              {{ $t('$sitePlaceholder') + ' (' + availableSites.length + ')' }}
```

Comment:
Current site count added as a debug helper, but this felt pretty nice

Reply:
I'm not sure if it's easy to understand what the number stands for

Reply:
I think it could be removed